src/reader_writer_interface.jl

```diff
@@ -43,14 +43,16 @@ abstract type AbstractWriter end
 # Reader functions
 
 """
-    PSRI.file_to_array(::Type{T}, path::String) where T <: AbstractReader
+    PSRI.file_to_array(::Type{T}, path::String, use_header::Bool = true, header::Vector{String} = String[]) where T <: AbstractReader
```

Comment:
I would advise to make these as keyword arguments instead of positional ones. Also I think that the default behaviour in our case if to set them as `false`. @joaquimg might have a different idea here.
```julia`
PSRI.file_to_array(::Type{T}, path::String; use_header::Bool = false, header::Vector{String} = String[])
``

Reply:
At first I also thought `false` would be a better default value. However, thinking a bit about it, I think `false` can be dangerous. Someone using the package may not be aware of the fact that the graf and the database do not necessarily have the same order of agents. Leaving `true` as default, it is required to pass the header or explicitly say that it wants to read the order of the graf file.

Reply:
Agreed! We can leave it as true then.

Reply:
`true` seems fine. Keywords should be used instead of positional indeed.